app.css

```diff
@@ -0,0 +1,88 @@
+html {
+  box-sizing: border-box;
+}
+*,
+*:before,
+*:after {
+  box-sizing: inherit;
+}
+html, 
+body {
+  font-size: 18px;
+}
+body {
+  font-family: Helvetica Neue, Helvetica, Arial, Sans-Serif;
```

Comment:
I believe it should be `sans-serif`, though it might not actually break anything